src/coniferest/session/__init__.py

```diff
@@ -186,6 +195,13 @@ def run(self) -> "Session":
     def terminate(self) -> None:
         self._terminated = True
 
+    def argsorttopk(self) -> np.ndarray:
```

Comment:
Consider to rename to `argtopk_scores(self, k: int) -> np.ndarray` and explicitly pass `k=self._sort_size`